scripts/apps/authoring/views/authoring-header.html

```diff
@@ -527,7 +527,7 @@
                 order="{{ editor[field._id].order }}"
                 ng-attr-title="{{schema[field._id].readonly === true ? readOnlyLabel : ''}}"
                 sd-validation-error="error[field._id.toLowerCase()]">
-                <label class="authoring-header__item-label">{{field.display_name}}</label>
+                <label class="authoring-header__item-label" ng-style="template.data.extra.DueBy !== null && {'margin-bottom':'35px'}">{{field.display_name}}</label>
```

Comment:
why is that extra margin needed? Could we add it without hardcoding the field name ID?

Reply:
> 

When Comin Up field switch is ON then due to message display alignment between Coming Up label and switch is mismatched that's why to match that sequence extra margin provided

Reply:
> 
![comingUp](https://user-images.githubusercontent.com/10600741/143451541-553539a3-bbfb-40a6-8a65-0b59519af9e4.png)

Reply:
This is with that extra CSS that fixes it? I was wondering how it looked before the fix

Reply:
> 

No, This scenario is without extra CSS

Reply:
I see. There are 2 problems with your solution
* hardcoding of `dueDate` field - it means that it would break if date time field was configured with another ID
* margin is hardcoded in pixels. Even the same text might take more or less lines in another language and the design would break

You could avoid both issues by adding `align-items: start;` to the parent element(with class "authoring-header__item").